manifests/v/valinet/ExplorerPatcher/2200.348.39.1/valinet.ExplorerPatcher.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: valinet.ExplorerPatcher
+PackageVersion: 2200.348.39.1
```

Comment:
```suggestion
PackageVersion: 22000.348.39.1
```

Reply:
Thanks; Looks like I missed a `0`; Should be fixed now